### ISSUE TITLE
Question: Max_iter and batch_size

### ISSUE THREAD

**Issue by lemonviv** [FederatedAI/FATE#4531]
Hi, in the algorithm [param](https://github.com/FederatedAI/FATE/blob/master/python/federatedml/param/logistic_regression_param.py) of hetero_lr, is the meaning of `iter` the same as `epoch`? For example, if the number of samples is 800 in the training dataset, and I set `max_iter = 20` and `batch_size = 100`, will the algorithm run `8` mini-batch executions in each iteration? I looked into the batch generator but got a bit confused. Appreciate it if any of you could help clarify it. Thanks in advance!

**Comment by talkingwallace** [comment]
Yes, your understanding is correct, you can also find descriptions for parameters in the document, or in /federatedml/param/logistic_regression_param.py.
